tests/integration/test_tostac.py

```diff
@@ -11,10 +11,10 @@
 from tests.integration.common import run_prepare_cli
 
 TO_STAC_DATA: Path = Path(__file__).parent.joinpath("data/tostac")
-ODC_METADATA_FILE: str = "ga_ls8c_ard_3-1-0_088080_2020-05-25_final.odc-metadata.yaml"
+ODC_METADATA_FILE: str = "ga_ls8c_ard_3-2-0_088080_2020-05-25_final.odc-metadata.yaml"
 STAC_TEMPLATE_FILE: str = "ga_ls_ard_3_stac_item.json"
 STAC_EXPECTED_FILE: str = (
-    "ga_ls8c_ard_3-1-0_088080_2020-05-25_final.stac-item_expected.json"
+    "ga_ls8c_ard_3-2-0_088080_2020-05-25_final.stac-item_expected.json"
```

Comment:
This refers to the test data files inside the repository, so the change to them is breaking Travis. We could rename the test data too, but I'd just revert these lines as it's the version the test data was made from.

Reply:
no problem. I'll revert the commits.